test/provisioner/provision.go

```diff
@@ -62,6 +70,42 @@ type InstallOverlay interface {
 	Edit(ctx context.Context, cfg *envconf.Config, properties map[string]string) error
 }
 
+func NewKeyBrokerService(provider string) (*KeyBrokerService, error) {
+	// Clone kbs repo
+	repoURL := "https://github.com/confidential-containers/kbs"
+	cmd := exec.Command("git", "clone", repoURL)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	err := cmd.Run()
+	if err != nil {
+		fmt.Printf("Error running git clone: %v\n", err)
+		return nil, err
+	}
+
+	// Create secret
+	content := []byte("This is my super secret")
```

Comment:
We need something unique to test if this is what we put in every test and this is something we get as output in every test, something like cluster name or something.